patches/minecraft/net/minecraft/client/network/play/ClientPlayNetHandler.java.patch

```diff
@@ -46,7 +46,7 @@
 +            net.minecraftforge.entity.PartEntity<?>[] aenderdragonpartentity = livingentity.getParts();
  
              for(int i = 0; i < aenderdragonpartentity.length; ++i) {
-                aenderdragonpartentity[i].func_145769_d(i + p_147281_1_.func_149024_d());
+                aenderdragonpartentity[i].func_145769_d((i + 1) + p_147281_1_.func_149024_d());
```

Comment:
Should have comment linking to the mc issue this fixes (E.g. ` //Forge: Fixes MC-123456`).